posts/views.py

```diff
@@ -69,6 +81,13 @@ def post_create(request):
         save_hashtags(parsedString['hashtags'], post)
     return redirect(request.META['HTTP_REFERER'] or 'landingpage')
 
+def set_post_replies(post):
```

Comment:
This causes the error written down here: https://github.com/7Pros/circuit/pull/142#issuecomment-114546313